runtime/src/prioritization_fee.rs

```diff
@@ -29,6 +41,24 @@ impl PrioritizationFeeMetrics {
         saturating_add_assign!(self.total_update_elapsed_us, val);
     }
 
+    fn update_prioritization_fee(&mut self, fee: u64) {
+        if fee == 0 {
+            saturating_add_assign!(self.non_prioritized_transactions_count, 1);
+            return;
+        }
+
+        // update prioritized transaction fee metrics.
+        saturating_add_assign!(self.prioritized_transactions_count, 1);
+
+        if fee > self.max_prioritization_fee {
+            self.max_prioritization_fee = fee;
+        }
+
+        if self.min_prioritization_fee.is_none() || fee < self.min_prioritization_fee.unwrap() {
+            self.min_prioritization_fee = Some(fee);
+        }
+    }
```

Comment:
similarly, potentially cleaner as
```suggestion
        self.min_prioritization_fee = self
            .min_prioritization_fee
            .map_or(fee, |min_fee| min_fee.min(fee));
```

Reply:
TIL about `map_or`, thanks!